### DISCUSSION TITLE
Plot gradient norms along training?

### DISCUSSION THREAD

**Discussion by GeoffNN** [apache/incubator-mxnet#21120] [Q&A]
Hi,

I'm trying to plot the overall gradient norm during training. I can't find how to access the gradient values.
Suppose I have a forward pass/loss defined, and I have this, using a `gluon.Trainer`. This training is distributed over multiple gpus.

```python
loss.backward()
trainer.step(cur_batch_size, ignore_stale_grad=ignore_stale_grad)
```

How can I get gradients to compute the sum of their L2 squared norms?

Thanks
